Specifications/QIR/Data-Types.md

```diff
@@ -294,5 +319,8 @@ There are special runtime functions defined for allocating or releasing an
 array of qubits.
 See [here](Quantum-Runtime.md#qubit-management-functions) for these functions.
 
+For all of these functions other than `reference` or `unreference`, if an
```

Comment:
What should happen if ref/unref on an array of qubits? What if the ref-count is > 1 but release_qubit_array is called? Basically, does an array of qubits own the qubits it contains or not? I'd propose it does _not_ and is subject to all array operations, leaving it to the clients to ensure they don't concatenate arrays of mismatched types, and that qubits must be released independent of the array's memory management.

Reply:
Addressed by Bettina's coming PR.